docs/RelayAPI.md

```diff
@@ -1,376 +1,504 @@
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+|user|string|YES|Address of user|
 ### Response
 ```javascript
-"12.1009234"
+1000 // total amount user can spend in a currency network
 ```
 
-## Relay transaction
-Sends a signed transaction
-
-Remove txId as response instead compute on client
+---
 
-`POST /relay`
-
-### Request
-```javascript
-// rawTransaction
+### Spendable amount to other user in currency network
+### `/networks/:network/users/:userA/spendables/:userB`
+Returns amount user A can spend to user B in a currency network.
+#### Request
 ```
-
+GET /networks/:network/users/:userA/spendables/:userB
+```
+#### Parameters
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+|userA|string|YES|Address of user A|
+|userB|string|YES|Address of user B|
 ### Response
 ```javascript
-"txId"
+90 // amount user A can spend to user B in a currency network
 ```
 
-# Exchange API
-
-## `TODO` Get balances
-Returns balances of user
-
-`GET networks/:networkAddress/users/:userAddress/exchanges/:currency`
+---
 
+### Transfer path in currency network
+### `/networks/:network/path-info`
+Returns the cheapest path, the estimated gas costs and maximal fees for a transfer.
+#### Request
+```
+POST /networks/:network/path-info
+```
 ### Parameters
-- `currency` -
-
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+### POST body
+|Name|Type|Required|Description|
+|-|-|-|-|
+|from|string|YES|Address of user who sends transfer|
+|to|string|YES|Address of user who receives transfer|
+|value|int|YES|Transfer amount in smallest unit|
+|maxFees|int|NO|Upper bound for transfer fees|
+|maxHops|int|NO|Upper bound for hops in transfer path|
 ### Response
 ```javascript
 {
-    "EUR": {
-        "available": "1000", // available trustlines money
-        "onOrders": "123", // amount on orders
-        "ethValue": "1.12" // exchange value
-    },
-    "ETH": {
-        "available": "123.112",
-        "onOrders": "1.1112",
-        "eurValue": "300"
-    }
+	"path": [
+    "0xcbF1153F6e5AC01D363d432e24112e8aA56c55ce", // from address
+		..., // hop addresses in path
+    "0x7Ec3543702FA8F2C7b2bD84C034aAc36C263cA8b" // to address
+  ],
+  "fees": 2, // maximal transfer fees
+  "estimatedGas": 76324 // estimated gas costs for transfer
 }
 ```
 
-## `TODO` Get deposits
-Returns deposit history within range `start` to `end`
-
-`GET networks/:networkAddress/users/:userAddress/exchanges/:currency/deposits?start=:start&end=:end`
+---
 
-### Parameters
-- `start` - UNIX timestamp for range start
-- `end` - UNIX timestamp for range end
-
-### Response
+### All events in currency network
+### `/networks/:network/events`
+Returns a list of event logs in a currency network.
+#### Request
+```
+GET /networks/:network/events?type=:type&fromBlock=:fromBlock
+```
+#### Parameters
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
```

Comment:
I'm am not sure if we should mix them like that. Network is a part of the path and I would not call it parameter. Are they referred to as parameter somewhere?

Reply:
Hmm get your point... But if I look at other REST API docs they also call it parameters. Some are distinguishing between **URL Parameters** and **Data Parameters**. Shall we do that also?

Reply:
Can you point me to an example?

Reply:
- This is just gist I found where they distinguish between **URL** and **Data** params: https://gist.github.com/iros/3426278
- Infura for example calls both just **Parameters** https://infura.io/docs/api/get/symbolFull
- 0x relayer api calls it **Parameters** and **Payload** https://github.com/0xProject/standard-relayer-api/blob/master/http/v0.md#get-v0token_pairs
- ...

Reply:
Yeah. A distinction would be good. And as I said we need both documentation and example. Like it is in the infura doc.